frontend/src/App.vue

```diff
@@ -1,48 +1,17 @@
 <template lang="pug">
-  v-app
-    v-app-bar(app clipped-left color="primary" height="100")
-      v-app-bar-nav-icon(@click="drawer = !drawer")
-      v-col
-        v-row TA MANAGEMENT PORTAL
-        v-row Department of Electrical Engineering,IIT JODHPUR
-        div(class="flex-grow-1")
-        div(class="flex-1")
-        div(class="flex-1")
-          v-avatar(color=" ")
-            v-icon(mdi-account-circle)
-      v-col
-        v-row(col="2" offset-md="4") Name
-        v-row(col="2") Teaching Assistant
-    v-navigation-drawer(v-model="drawer" app clipped color="grey lighten-4")
-      v-list(dense class="grey lighten-4")
-        template(v-for="(item, i) in items")
-          v-row(v-if="item.heading" :key="i" align="center")
-            v-col(cols="6")
-              v-subheader(v-if="item.heading") {{ item.heading }}
-          v-divider(v-else-if="item.divider" :key="i" dark class="my-4")
-          v-list-item v-else :key="i" @click=""
-            v-list-item-action
-              v-icon({{ item.icon }})
-            v-list-item-content
-              v-list-item-title(class="grey--text") {{ item.text }}
-    v-content
-      router-view
+    v-app
+        v-content
```

Comment:
`HeaderToolbar` abstraction looks good but i think it should be kept in `App.vue` only and not included in every view it is required in. Instead you can render in conditionally with `v-if`

Reply:
done

Reply:
done